opentreemap/manage_treemap/views/__init__.py

```diff
@@ -47,6 +51,8 @@ def _update_instance_fields(json_data, instance, validation_fn=None):
         try:
             validate_is_public(instance)
             instance.save()
+            if should_update_eco_rev:
+                instance.update_eco_rev()
```

Comment:
You should also update the `universal_rev` here: see https://github.com/OpenTreeMap/otm-core/blob/develop/opentreemap/treemap/instance.py#L237-L241 for an explanation

Reply:
Updated. Thanks.